README.md

```diff
@@ -53,15 +53,15 @@ assert_eq!(edn[3]["false"], Edn::Key("f".to_string()));
  #![recursion_limit="512"]
  #[macro_use] extern crate edn_rs;
  
- use std::collections::{HashMap, HashSet};
+ use std::collections::{BTreeMap, BTreeSet};
  use crate::edn_rs::serialize::Serialize;
  
  fn main() {
      ser_struct!{
```

Comment:
Guess this could be kept as HashMaps and HashSets as they depend only on the serialization module.
Maybe add an example with BTreeMap and BTreeSet

Reply:
I will put one field for each, so 4 of them, Hash{Map, Set} and BTree{Map, Set}

Reply:
Solved, by you!